### ISSUE TITLE
model its invisibile

### ISSUE THREAD

**Issue by C3scoo** [Godlander/objmc#78]
i have a 512 X 512 texture and the png work fine in blender but i dont see model how to fix?

**Comment by kuraiskrap** [comment]
no idea if its related or not, check errors in your log, i had that issue on a fork and had the error
Missing textures in model minecraft:potion#inventory:
 minecraft:textures/atlas/blocks.png:minecraft:item/whatever

**Comment by kleo-dev** [comment]
Yeah same here, i checked the logs, but it's only for inventory stuff, i got the same warning with my other non obj models but they worked just fine, @Godlander please help,

I might be able to fix this myself, but that's if @Godlander doesn't respond

**Comment by Godlander** [comment]
if the sample pack is working, then you did something wrong.

if the sample pack is not working, then it is an incompatible version of the game/pack.

objmc has not been updated for 1.21.5 or above, so it will not work with latest version.
